environments/development.env

```diff
@@ -6,7 +6,7 @@ NODE_ENV=development
 UBUNTU_SSO_URL=https://login.ubuntu.com
 OPENID_VERIFY_URL=http://localhost:8000/login/verify
 LP_API_URL=https://api.launchpad.net
-WEBHOOK_ENDPOINT=https://build.staging.snapcraft.io/webhook/notify
+WEBHOOK_ENDPOINT=http://localhost:8000/api/webhook/notify
```

Comment:
Could you add a suitable `WEBHOOK_ENDPOINT` to the staging and production environments as well?  Looks like they should be `https://build.staging.snapcraft.io/api/webhook/notify` and `https://build.snapcraft.io/api/webhook/notify` respectively.